lib/NMSmodel/ceinms/MTU.cpp

```diff
@@ -318,6 +321,48 @@ namespace ceinms {
             cos(pennationAngleAtT);
     }
 
+    template<typename Activation, typename Tendon, CurveMode::Mode mode>
+    void MTU<Activation, Tendon, mode>::updateMtuStiffness() {
+
+        double optimalFiberLengthAtT = optimalFibreLength_*(percentageChange_*(1.0 - activation_) + 1);
+        ////:TODO: strong review with the code... lot of check for closeness to 0
+        double normFiberLengthAtT = fibreLength_ / optimalFiberLengthAtT;
+        double normFiberLength = fibreLength_ / optimalFibreLength_;
+        double fiberVel = fibreVelocity_ / optimalFibreLength_;
+        if (fiberVel > maxContractionVelocity_)
+            fiberVel = maxContractionVelocity_;
+        if (fiberVel < -maxContractionVelocity_)
+            fiberVel = -maxContractionVelocity_;
+        double normFiberVelocity = fiberVel / maxContractionVelocity_;
```

Comment:
May of these lines are similar to `updateMuscleForce`. Do you think it would be possible to factorise a bit?